CursedSnake/ViewsAndScenes/GameScene.swift

```diff
@@ -223,7 +223,8 @@ class GameScene: SKScene, SKPhysicsContactDelegate {
         }
         else if touchedNode.name == "returnToMenu" {
             let vc = UIHostingController(rootView: TitleView())
-            view?.presentScene(vc)
+            // TODO: are there any edge cases when wither view or window are nil?
```

Comment:
either*

Reply:
I don't think so? I'll take another look

Reply:
> either*

I don't think so? I'll take another look

Reply:
ok then just leave the force unwraps and hope it doesn't crash